scipy/stats/_continuous_distns.py

```diff
@@ -1252,6 +1253,10 @@ class chi2_gen(rv_continuous):
 
     %(after_notes)s
 
+    See Also
```

Comment:
According to the docstring standard, the "See Also" section precedes the "Notes" section, so this should be moved up.

Reply:
Oops, I guess I followed asymmetric laplace. I fixed these three in one commit and the remaining nine in a separate commit in case we don't want others to make the same mistake.